sycl/source/detail/queue_impl.cpp

```diff
@@ -26,7 +26,9 @@
 namespace sycl {
 inline namespace _V1 {
 namespace detail {
-std::atomic<unsigned long long> queue_impl::MNextAvailableQueueID = 0;
+// Treat 0 as reserved for "host" queue
```

Comment:
I think this should be reworded as something like "reserved for host tasks" to avoid confusion with the host queues we used to have.